back/src/Guide/Database/Convert.hs

```diff
@@ -0,0 +1,173 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+
+module Guide.Database.Convert
+       (
+       -- Encoders
+         boolParam
+       , boolParamNullable
+       , textParam
+       , textParamNullable
+       , traitTypeParam
+       , uidParam
+       -- Decoders
+       , textArrayColumn
+       , textDecoderNullable
+       , textColumn
+       , timestamptzColumn
+       , uidColumn
+       , categoryRow
+       , itemRow
+       , traitRow
+       ) where
+
+
+import Imports
+
+import Data.Functor.Contravariant (contramap)
+
+import qualified Data.Map as Map
+import qualified Data.Set as Set
+import qualified Data.Text as T
+import qualified Hasql.Decoders as HD
+import qualified Hasql.Encoders as HE
+
+import Guide.Markdown (toMarkdownBlock, toMarkdownInline, toMarkdownTree)
+import Guide.Types.Core (Category (..), CategoryStatus(..), Item (..), ItemSection(..), Trait (..),
+                         TraitType (..))
+import Guide.Utils (Uid (..))
+
+
+-- Convert functions for database
```

Comment:
This should be a module haddock (`-- |` before `module`), not a stray comment after imports.